packages/esm-patient-list-app/src/offline-patient-table/offline-patient-table.component.tsx

```diff
@@ -0,0 +1,226 @@
+import React from 'react';
+import DataTable, {
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableHeader,
+  TableRow,
+  FilterRowsData,
+  TableSelectAll,
+  TableSelectRow,
+} from 'carbon-components-react/lib/components/DataTable';
```

Comment:
Same here; I think all of these `/lib` references may be `.../es/` ones. Maybe this is a general repo problem. Would be superb if you could check this!